packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -6,13 +6,27 @@ import logger from '@wdio/logger'
 
 const log = logger('@wdio/browserstack-service')
 
+type BrowserstackLocal = BrowserstackLocalLauncher.Local & {
+    pid?: number;
+    stop(callback: (err?: any) => void): void;
+}
+
+type ExtendedCapabilities = WebDriver.Capabilities & {
+    'browserstack.local'?: boolean;
+}
+
+type Capabilities = (ExtendedCapabilities | ExtendedCapabilities[])
+
 export default class BrowserstackLauncherService {
```

Comment:
Let's ensure we implement according to the service standard:
```suggestion
export default class BrowserstackLauncherService implements WebdriverIO.ServiceInstance {
```